hikari/impl/special_endpoints.py

```diff
@@ -1093,6 +1094,12 @@ class CommandBuilder(special_endpoints.CommandBuilder):
     _id: undefined.UndefinedOr[snowflakes.Snowflake] = attr.field(default=undefined.UNDEFINED, kw_only=True)
     _default_permission: undefined.UndefinedOr[bool] = attr.field(default=undefined.UNDEFINED, kw_only=True)
 
+    _name_localizations: typing.Optional[typing.Dict[Locale | str, str]] = attr.field(default=None, kw_only=True)
```

Comment:
Why doesn't this default to `undefined.UNDEFINED`?

Reply:
So should I also use `undefined.UndefinedOr` instead of `typing.Optional`?

Reply:
Yeah